lib/zold/upgrades.rb

```diff
@@ -0,0 +1,27 @@
+require 'semantic'
+
+module Zold
+  # Class to manage data upgrades (when zold itself upgrades).
+  class Upgrades
+    def initialize(version_file)
+      @version_file = version_file
+    end
+
+    # @todo #285:30min Write the upgrade manager tests that ensure:
+    #  - Nothing breaks without the version file.
+    #  - The upgrade scripts run when there is a version file and there are pending upgrade scripts.
+    #  - Make sure *only* the correct upgrade scripts run.
+    def run
+      from_version = @version_file.version
+      return unless from_version
+      upgrade(from_version)
```

Comment:
@botanicus there is something wrong with this logic... I believe, this design would be more natural:

```
def run
  scripts.each do |v|
    @version.apply(v) do
      # run the script by the name "upgrades/#{v}.rb"
    end
  end
end
```

`scripts` is a private method that returns all scripts availble in `upgrades` directory as a sorted array of strings.

Thus, `@version.apply` will decide whether to apply on not, according to its internal information and will update the content of the file.

WDYT?